code/game/objects/machinery/mortar.dm

```diff
@@ -231,6 +238,18 @@
 		addtimer(CALLBACK(GLOBAL_PROC, .proc/playsound, T, fall_sound, 50, 1), travel_time)
 		addtimer(CALLBACK(src, .proc/detonate_shell, T, mortar_shell), travel_time + 45)//This should always be 45 ticks!
 
+	if(istype(I, /obj/item/ai_target_beacon))
+		for(var/mob/living/silicon/ai/AI AS in GLOB.ai_list)
```

Comment:
Should not be a for, because that code doesn't work if there are multiple ais.

Also, does this work if there are no ais ?